modules/sno-du-subscribing-to-the-operators-needed-for-platform-configuration.adoc

```diff
@@ -0,0 +1,83 @@
+// Module included in the following assemblies:
+//
+// *scalability_and_performance/sno-du-deploying-clusters-on-single-nodes.adoc
+
+:_content-type: PROCEDURE
+[id="sno-du-subscribing-to-the-operators-needed-for-platform-configuration_{context}"]
+= Subscribing to the Operators
+
+The subscription provides the location to download the Operators needed for platform configuration.
+
+.Procedure
+
+* Use the following example to configure the subscription:
++
+[source,yaml]
+----
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: cluster-logging
+  namespace: openshift-logging
+spec:
+  channel: "stable" <1>
+  name: cluster-logging
+  source: redhat-operators
+  sourceNamespace: openshift-marketplace
+  installPlanApproval: Manual <2>
+---
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: local-storage-operator
+  namespace: openshift-local-storage
+spec:
+  channel: "stable" <3>
+  installPlanApproval: Automatic
+  name: local-storage-operator
+  source: redhat-operators
+  sourceNamespace: openshift-marketplace
+  installPlanApproval: Manual
+---
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: performance-addon-operator
+  namespace: openshift-performance-addon-operator
+spec:
+  channel: "4.10" <4>
+  name: performance-addon-operator
+  source: performance-addon-operator
+  sourceNamespace: openshift-marketplace
+  installPlanApproval: Manual
+---
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+    name: ptp-operator-subscription
+    namespace: openshift-ptp
+spec:
+  channel: "stable" <5>
+  name: ptp-operator
+  source: redhat-operators
+  sourceNamespace: openshift-marketplace
+  installPlanApproval: Manual
+---
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: sriov-network-operator-subscription
+  namespace: openshift-sriov-network-operator
+spec:
+  channel: "stable" <6>
+  name: sriov-network-operator
+  source: redhat-operators
+  sourceNamespace: openshift-marketplace
+  installPlanApproval: Manual
+----
+<1> Specify the channel to get the `cluster-logging` Operator.
+<2> Specify `Manual` or `Automatic`. In `Automatic` mode, the Operator automatically updates to the latest version in the channel as they become available in the registry. In `Manual` mode, new Operator versions are installed only after they are explicitly approved.
```

Comment:
s/the latest version in the channel as they become available/
the latest versions in the channel as they become available  --or--
the latest version in the channel as it becomes available

Reply:
changed.